reframe/frontend/cli.py

```diff
@@ -32,7 +32,18 @@ def format_check(check, detailed):
             '      - description: %s' % check.descr,
             '      - systems: %s' % ', '.join(check.valid_systems),
             '      - environments: %s' % ', '.join(check.valid_prog_environs),
-            '      - modules: %s' % ', '.join(check.modules),
+            '      - modules: %s' % ', '.join(check.modules)
+        ]
+
+        if check.num_tasks <= 0:
+            num_tasks_per_node = check.num_tasks_per_node or 1
+            min_num_tasks = (-check.num_tasks if check.num_tasks else
+                             num_tasks_per_node)
+            num_nodes_required = min_num_tasks // num_tasks_per_node
+            lines.append(
+                '      - required nodes: %s' % num_nodes_required)
```

Comment:
Here we duplicating the same logic we have internally. I don't think it makes sense to be so precise, because for example, you don't print the required nodes for a non-flexible test. I suggest being simple and showing only if the test is flexible or not. To check the number of nodes, the user has to look into the check as is the case for every other check.